### PR TITLE
Fix immediate decoding for lwsp and flwsp (#289)

### PR DESCRIPTION
**Pull request by kpgriesser**
This fixes bug #289 in which an lwsp instruction execution had an incorrect immediate field calculation. The root cause was due to missing the '0b' prefix on a binary constant.  I also fixed flwsp which had the same problem ( but does not have a test ).

Note: Only lines 446 and 460 in RevCore.cc changed. The rest are auto-formatting changes.

### CODE REVIEW

**Review by leekillough — APPROVED** [review]
LGTM

**Review by donofrio — APPROVED** [review]
Great find! Looks good to merge in

**Review by rkabrick — APPROVED** [review]
LGTM!

File: test/lwsp/Makefile [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Python configuration file
+# REVCFG ?= ./rev-memh.py
+REVCFG ?= ../rev-model-options-config.py
+
+# Rev toolchain
+CC=riscv64-unknown-elf-g++
```
Comment (leekillough) [inline test/lwsp/Makefile:6]: This should be `CXX=${RVCXX}` once my [PR](https://github.com/tactcomplabs/rev/pull/291) is merged.

`CC` should only be used for C compilers; `CXX` for C++ compilers.

You may not need to create your own `Makefile` -- `test/CMakeLists.txt` has a general way of handling tests.

Comment (kpgriesser) [reply test/lwsp/Makefile:6]: thanks - I'll move to the general build method.

Comment (leekillough) [reply test/lwsp/Makefile:6]: It is already in my [PR](https://github.com/tactcomplabs/rev/pull/291)

**Pull request merged**